driver-mongodb/src/main/resources/activities/mongodb-basic.yaml

```diff
@@ -0,0 +1,57 @@
+# nb -v run driver=mongodb yaml=mongodb-basic connection=mongodb://127.0.0.1 database=testdb tags=phase:rampup
```

Comment:
There is now a `description` property supported in YAMLs that hasn't been documented yet. You might want to add a `description:` to this yaml.

Reply:
Updated. :ok_hand: